### PR TITLE
redshift: fix disable mac-specific configs on linux platform

### PR DESCRIPTION
**Pull request by emanresusername**
- [x] Have you followed the [guidelines for contributing](https://github.com/Linuxbrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Linuxbrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by emanresusername** [comment]
didn't try the `--build-bottle` thing, looks similar to [this issue](https://github.com/Linuxbrew/homebrew-core/issues/4499#issue-261680393) and redshift is mentioned there too... will try to incorporate that ...

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -32,7 +32,13 @@ First time contributing to Homebrew? Read our [Code of Conduct](https://github.c
...
 * `brew edit foo` and make edits
 * leave the [`bottle`](http://www.rubydoc.info/github/Homebrew/brew/master/Formula#bottle-class_method) as-is
-* `brew uninstall --force foo`, `brew install --build-from-source foo`, `brew test foo`, and `brew audit --strict foo`
+* 
+    ```
+    brew uninstall --force foo
+    brew install --build-from-source foo
+    brew test foo
+    brew audit --strict foo
+   ```
```
Comment (iMichka) [inline CONTRIBUTING.md]: That part is actually to be changed in the upstream repository first: https://github.com/Homebrew/homebrew-core, and the I'll merge it here later. We want to try to stay as close as possible as upstream.

Also, this change is unrelated to this pull request.

**Comment by emanresusername** [comment]
squashed

**Comment by iMichka** [comment]
@emanresusername Thanks for your contribution!

**Pull request closed without merging**